charts/kong/templates/deployment.yaml

```diff
@@ -134,7 +134,15 @@ spec:
           {{- if .hostPort }}
           hostPort: {{ .hostPort }}
           {{- end}}
-          protocol: TCP
+          protocol: {{ .protocol }}
+        {{- end }}
+        {{- range .Values.udpProxy.stream }}
+        - name: stream-udp-{{ .containerPort }}
+          containerPort: {{ .containerPort }}
+          {{- if .hostPort }}
+          hostPort: {{ .hostPort }}
```

Comment:
targetPort is missed here.

Reply:
This is the Deployment container template; it only includes the target port.

We use target and container port interchangeably, since they're similar things with different names depending on whether you're building a Service or Deployment. We populate the Service targetPort value at https://github.com/Kong/charts/blob/bd8e8600f708a956e4041ec84d6a0814e1fbcfd2/charts/kong/templates/_helpers.tpl#L140

Reply:
understand. the changes lgtm.